### ISSUE TITLE
Dependencies: install browser-env and other housekeeping

### ISSUE THREAD

**Issue by cake-pie** [openscope/openscope#1752]
### install browser-env
Some classes have tests omitted, or commented out and marked todo, because the class imports jQuery and in the absence of a browser environment we get `jQuery requires a window with a document`.

ava docs has a [recipe](https://github.com/avajs/ava/blob/master/docs/recipes/browser-testing.md) that looks like it should solve this.

---
### housekeeping

If it's used only in `./test` and/or `./tools` but not `./src` then it should be in `devDependencies` and not `dependencies`:
```
coveralls
fancy-log
mkdirp
showdown
yargs
```

Also, are these still used somewhere?
```
cli-table
glob-fs
```

**Comment by n8rzz** [comment]
you are correct, `devDependencies` should be listed as such in the `package.json`.  However, we deploy via Heroku and build on the box there.  I haven't looked at this since I set it up probably 5 years ago now.  Maybe it's better now?  

TL;DR `devDependencies` are included in `dependencies` on purpose so that we could run the build commands on our (heroku) server during the deployment process

**Comment by cake-pie** [comment]
Looks like it should be better now.

via [here](https://devcenter.heroku.com/articles/nodejs-support#package-installation)

> By default, Heroku will install all dependencies listed in `package.json` under `dependencies` and `devDependencies`.
>
> After running the installation and build steps Heroku will strip out the packages declared under `devDependencies` before deploying the application.

This behavior was changed in [Mar 2018](https://devcenter.heroku.com/changelog-items/1376)

**Comment by n8rzz** [comment]
that pleases me
